### PR TITLE
Users

### PR DESCRIPTION
**Pull request by Hilder64**
Functionality regarding Users is ready. User model added. User service added. Exceptions in the repository and service are ready. Controller added.

### CODE REVIEW

File: App.Web/Modules.cs [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
     /// </summary>
     [ModuleUsing(typeof(Example.ExampleModule))] // < ---- Example of module registration
+    [ModuleUsing(typeof(Users.UsersModule))]
 
     [ModuleUsing(typeof(News.NewsModule))]
     [ModuleUsing(typeof(UserSupport.UserSupportModule))]
     [ModuleUsing(typeof(Goods.GoodsModule))]
     [ModuleUsing(typeof(Stocks.StocksModule))]
     [ModuleUsing(typeof(Loans.LoansModule))]
-    [ModuleUsing(typeof(Cards.CardsModule))]
```
Comment (ghost) [inline App.Web/Modules.cs:16]: Why cards and regularpayments modules were removed ?

**Review by ghost — CHANGES_REQUESTED** [review]

File: App.Web/Modules.cs (hunk 1 above)
Comment (Hilder64) [reply App.Web/Modules.cs:16]: When i pull repository, cards and regular payments projects was installed wrong in my local instance and my visual studio version. That changes were made for successful project startup in my machine.

Comment (Hilder64) [reply App.Web/Modules.cs:16]: @anton-beetroot-23071996 I did not find directory for App.RegualrPayments, but it is noticed in ApiTemplate.sln file. Lack of App.RegualrPayments sources i find in curent state of master branch in https://github.com/NikolenkoAnton/KPZ-PI-318-SUBGROUP_2 repository. Please pay attention.

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Жду вас на паре для защиты лабораторной работы

**Pull request merged**